libs/shared/db/mysql/account/src/lib/keysley-types.ts

```diff
@@ -55,8 +55,10 @@ export interface Accounts {
   emailVerified: Generated<number>;
   kA: Buffer;
   wrapWrapKb: Buffer;
+  wrapWrapKb2: Buffer | null;
   authSalt: Buffer;
   verifyHash: Buffer;
+  verifyHash2: Buffer | null;
```

Comment:
Same as above

Reply:
These have all be updated post fix '2' is now 'Version2' across the board.